### ISSUE TITLE
Review title to be limited to a line break (Enhancement)

### ISSUE THREAD

**Issue by arthurgouveia** [hugopl/reviewit#56]
In long commit messages that have line breaks on it's messages, I'd like to suggest getting only the first line of the commit to show up as a title and the following lines to be underneath.

Here's an example (screenshot): http://d.pr/i/1dbgs

**Comment by hugopl** [comment]
reviewit uses `git show -s --format="%s"` to generate the subject line, for some reason git is saying that the subject is the whole thing, ignoring the line break, maybe because you used a colon at the end of the line.

**Comment by hugopl** [comment]
yes... it's exactly this, try `git log --format=oneline` on this commit and you will see.

from git man page:
```
Though not required, it’s a good idea to begin the commit message with a single short (less than 50 character) line summarizing the change, followed by a blank line and then a more thorough description. The text up to the first blank line in a commit message is treated as the commit title, and that title is used throughout Git. For example, git-format-patch[1] turns a commit into email, and it uses the title on the Subject line and the rest of the commit in the body.
```

However I didn't find yet the rules git use to identify what's the subject.

I think a simple fix is just alert the user when the subject is greater than X (120?) characters, since a commit subject ending with ":" is also not good.

**Comment by hugopl** [comment]
ok.. even simpler... forget about git and just get the first line, no matter how long it's.

**Comment by hugopl** [comment]
Fixed in commit b369348ab3aefdb8e3aa5d0bdbb076b605b472e2.

Thanks.
